app/components/autocomplete/emoji_suggestion/index.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {bindActionCreators} from 'redux';
+import {connect} from 'react-redux';
+import {createSelector} from 'reselect';
+
+import {autocompleteUsers} from 'mattermost-redux/actions/users';
+import {getCustomEmojisByName} from 'mattermost-redux/selectors/entities/emojis';
+
+import {getTheme} from 'app/selectors/preferences';
+import {EmojiIndicesByAlias} from 'app/utils/emojis';
+
+import EmojiSuggestion from './emoji_suggestion';
+
+const getEmojisByName = createSelector(
+    getCustomEmojisByName,
+    (customEmojis) => {
+        const emoticons = [];
+        for (const [key] of [...EmojiIndicesByAlias.entries(), ...customEmojis.entries()]) {
+            emoticons.push(key);
+        }
+
+        return emoticons;
+    }
+);
+
+function mapStateToProps(state, ownProps) {
+    const {currentChannelId} = state.entities.channels;
+    const emojis = getEmojisByName(state);
+
+    let postDraft;
+    if (ownProps.isSearch) {
```

Comment:
we won't be using the emoji autocomplete in search so this is completely unnecessary